libusb/core.c

```diff
@@ -1151,11 +1167,31 @@ int API_EXPORTED libusb_get_max_iso_packet_size(libusb_device *dev,
  * \returns the same device
  */
 DEFAULT_VISIBILITY
-libusb_device * LIBUSB_CALL libusb_ref_device(libusb_device *dev)
+libusb_device *LIBUSB_CALL libusb_ref_device(libusb_device *dev)
```

Comment:
Why this change?

Reply:
Because I use clang format with the linux kernel style as that is what most code in libusb seems to use. If this whitespace bothers you I an revert it.